js/src/forum/components/UserPage.js

```diff
@@ -94,14 +95,33 @@ export default class UserPage extends Page {
     app.preloadedApiDocument();
 
     app.store.all('users').some(user => {
-      if ((user.username().toLowerCase() === lowercaseUsername || user.id() === username) && user.joinTime()) {
+      if ((user.username().toLowerCase() === lowercaseUsername) && user.joinTime()) {
         this.show(user);
         return true;
       }
     });
 
     if (!this.user) {
-      app.store.find('users', username).then(this.show.bind(this));
+      app.request({
```

Comment:
This `app.request()` can be replaced with `app.store.find('users', { filter: ... })`

Reply:
Changed!